### ISSUE TITLE
bigquery: INTERVAL not a recognized type

### ISSUE THREAD

**Issue by Thomasdezeeuw** [googleapis/google-cloud-go#4663]
**Client**

BigQuery

**Environment**

N/A

**Go Environment**

$ go version
go version go1.16.6 darwin/arm64
$ go env
```go
GO111MODULE=""
GOARCH="arm64"
GOBIN=""
GOCACHE="/Users/thomas/Library/Caches/go-build"
GOENV="/Users/thomas/Library/Application Support/go/env"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="arm64"
GOHOSTOS="darwin"
GOINSECURE=""
GOMODCACHE="/Users/thomas/go/pkg/mod"
GONOPROXY=""
GONOSUMDB=""
GOOS="darwin"
GOPATH="/Users/thomas/go"
GOPRIVATE=""
GOPROXY="https://proxy.golang.org,direct"
GOROOT="/opt/homebrew/Cellar/go/1.16.6/libexec"
GOSUMDB="sum.golang.org"
GOTMPDIR=""
GOTOOLDIR="/opt/homebrew/Cellar/go/1.16.6/libexec/pkg/tool/darwin_arm64"
GOVCS=""
GOVERSION="go1.16.6"
GCCGO="gccgo"
AR="ar"
CC="clang"
CXX="clang++"
CGO_ENABLED="1"
GOMOD="/Users/thomas/src/slight/sql-runner/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -arch arm64 -pthread -fno-caret-diagnostics -Qunused-arguments -fmessage-length=0 -fdebug-prefix-map=/var/folders/lv/qbz3lh7j6gdc43dd1p22bcc40000gn/T/go-build1478395030=/tmp/go-build -gno-record-gcc-switches -fno-common"

```

**Code**

e.g.
```go
package main

func main() {
        client, _ := bigquery.NewClient(ctx, "my-project-ID")
	q := client.Query("SELECT interval_field FROM some_table_with_interval_field")
        rows, _ := q.Run(..)
        var nullable_values []bigquery.Value
        err := rows.Next(&nullable_values)
        // The `error` above will be set.
}
```

**Expected behavior**

I expected the BigQuery client to support the `INTERVAL` type.

**Actual behavior**

Got an error "unrecognized type: INTERVAL", I think from https://github.com/googleapis/google-cloud-go/blob/1063c601a4c4a99217b45be0b25caa460e7157a1/bigquery/value.go#L950-L951

**Comment by shollyman** [comment]
Yeah, this is some pending feature work on my end.  I've mostly been trying to figure out how much I need to build towards supporting iso 8601 interval types, as time.Duration isn't sufficient for type mapping here..

**Comment by shollyman** [comment]
Apologies, lost track of this FR.

Interval support was released as part of the 1.32.0 version: 
https://github.com/googleapis/google-cloud-go/blob/main/bigquery/CHANGES.md#1320-2022-05-06
